Version 1.10 README and CHANGELOG (#484)
KxSystems/kx-vscode: README.md

  ![kdb results view](https://github.com/KxSystems/kx-vscode/blob/main/img/kdbview-results.png?raw=true)

## AxLibraries

The following features of [AxLibraries](https://code.kx.com/developer/libraries/) are integrated and can be used within the extension:

> - Grammar of Graphics

Reviewer: ```suggestion
- [Grammar of Graphics](#grammar-of-graphics)
```

> - q static linter

Reviewer: Do we need some more words on this, can it link to an existing section in the doc?

> **Note:** To use those features, AxLibraries should be installed folowing [installation instructions.](https://code.kx.com/developer/libraries/#installation)

Reviewer 2: ```suggestion
**Note:** To use those features, you should install AxLibraries following the [installation instructions.](https://code.kx.com/developer/libraries/#installation)
```

### Grammar of Graphics

> Calling `.qp.display` will display the plot locally.

Reviewer 2: ```suggestion
Calling `.qp.display` displays the plot locally.
```

> Calling `.qp.display` displays the plot locally.

Reviewer: @ecmel Should we have more detail here on what you need to do to get ggplots working?

Author: @LizNorris The link to the installation instructions are added in the Note section below.
